src/constants/societyNames.js

```diff
@@ -0,0 +1,8 @@
+const generateIdForSociety = societyName => ({
+  istelle: 1,
+  invictus: 2,
+  phoenix: 3,
```

Comment:
Apologies for the back and forth, but the name `generateIdForSociety` sounds a bit misleading. Why not `assignIdForSociety` since we are not generating but assigning? no? Also, wasn't the point of this to be the other way round such that when we get the 1 from the backend, we map it to a society string. This assumes we will pass the actual society names to the function.We want the numbers to be evaluated into the strings instead of vice versa. No?

Reply:
I think this would compromise readability i.e. getting an id/integer from the backend, what do you think? @kimani-ndegwa

Reply:
This is supposed to be a local map on the front-end that helps us evaluate against integers instead of strings. @benfluleck and I had discussed this. I believe he can give more contextual insight.